src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs

```diff
@@ -908,7 +917,7 @@ private bool WriteLocalFileHeader(bool isEmptyFile, bool forceWrite)
             Debug.Assert(_storedEntryNameBytes.Length <= ushort.MaxValue);
 
             // decide if we need the Zip64 extra field:
-            Zip64ExtraField zip64ExtraField = default;
+            Zip64ExtraField zip64ExtraField = new();
             bool zip64Used = false;
```

Comment:
Could we make `zip64ExtraField` nullable and replace `zip64Used` with a null check here too?

No comments besides that, all looks good to me thanks.

Reply:
Good catch. Fixed.